### ISSUE TITLE
Dockerfile issue

### ISSUE THREAD

**Issue by singhhp1069** [ipfs/go-ipfs#5918]
While running a dockerfile i am getting this error : 

+ cd /go/src/github.com/ipfs/go-ipfs
+ mkdir .git/objects
mkdir: cannot create directory '.git/objects': File exists

any specific reason

**Comment by Stebalien** [comment]
That's not a lot to go on. What exactly did you try, what OS, what versions, what's the full output, etc.

**Comment by Stebalien** [comment]
It looks like something is ignoring the `.dockerignore` file or your `$SRC_DIR` is dirty. Can you try removing any temporary files?

**Comment by Stebalien** [comment]
Wait, which source are you using?

**Comment by singhhp1069** [comment]
https://github.com/ipfs/go-ipfs#master

**Comment by Stebalien** [comment]
There's a `.dockerignore` file there.

**Comment by singhhp1069** [comment]
Yes, I did some modification and my docker not only running IPFS but there are few other services as well, I agree, I might have faced this issue because I removed the .dockeringore and did some add on.
